### PR TITLE
Create output panel

### PR DESCRIPTION
**Pull request by zmb3**
Convert the test panel into a general purpose output panel.

Connect build and test output to the output panel.

With this change the package is completely usable without linter, and it paves the way for displaying output for other events (for example, #582).

Here's a demonstration of the changes showing build and test failures with linter disabled.
Note that the ansi colors and clickable links from #599 are still supported.

output

### CODE REVIEW

**Comment by zmb3** [comment]
Update tools integration:

update

**Comment by lloiser** [comment]
Just a short review:
Can you please adjust the text for test lint (`go test -c`). I was really confused in the beginning that it "runs" the tests even though I've disabled it. Until I took a closer look at the actual text that says `-c` so it was the compilation/lint using `go test`.

**Comment by zmb3** [comment]
Sure, how about `Compiling tests`?

File: lib/test/test-panel-manager.js [hunk 1]
```diff
@@ -1,72 +0,0 @@
...
-    }
-    return this.emptyContent()
-  }
-
-  update (props) {
-    const oldProps = this.props
-    this.props = Object.assign({}, oldProps, props)
-
-    if (this.props.exitcode === 124) {
-      const timeoutMessage = 'Tests timed out after ' + atom.config.get('go-plus.test.timeout') + 'ms'
```
Comment (lloiser) [inline lib/test/test-panel-manager.js]: This is gone but it was quite useful. Now an empty panel pops up if this happens.

**Comment by lloiser** [comment]
👍  maybe also include the full command (in a smaller font-size?)

**Comment by joefitzgerald** [comment]
Let's not have different file sizes....

**Pull request merged**